test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -631,33 +620,25 @@ public void testGetDocContent() throws Exception {
     Map<String, String> configEntries = new HashMap<String, String>();
     configEntries.put("db.user", "sa");
     configEntries.put("db.password", "");
-    configEntries.put("db.url",
-        "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=");
-    configEntries.put("db.everyDocIdSql", "select * from employees");
+    configEntries.put("db.url", JdbcFixture.URL);
+    configEntries.put("db.everyDocIdSql", "select * from data");
     configEntries.put("db.singleDocContentSql",
-        "select * from employees where ID = ?");
+        "select * from data where ID = ?");
     configEntries.put("adaptor.namespace", "Default");
     configEntries.put("db.modeOfOperation", "rowToText");
-    configEntries.put("db.metadataColumns",
-        "ID:col1, FIRST_NAME:col2, LAST_NAME:col3");
+    configEntries.put("db.metadataColumns", "ID:col1, NAME:col2");
 
     final Config config = createStandardConfig(configEntries);
     DatabaseAdaptor adaptor = new DatabaseAdaptor();
     adaptor.init(TestHelper.createConfigAdaptorContext(config));
 
     MockRequest request = new MockRequest(new DocId("1001"));
-    MockResponse response = new MockResponse();
+    RecordingResponse response = new RecordingResponse();
     adaptor.getDocContent(request, response);
 
-    assertEquals("col1", adaptor.metadataColumns.getMetadataName("ID"));
-    assertEquals("col2", adaptor.metadataColumns.getMetadataName("FIRST_NAME"));
-    assertEquals("col3", adaptor.metadataColumns.getMetadataName("LAST_NAME"));
-
-    assertEquals("1001", response.metadata.getOneValue(adaptor.metadataColumns
+    assertEquals("1001", response.getMetadata().getOneValue(adaptor.metadataColumns
```

Comment:
What I was asking is why this test has a completely different set of assertions on the metadata from the getDocIds test. That is, why not just build an expected Metadata object here and assert that it matches response.getMetadata()?